unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -92,5 +109,27 @@ void SetMinimapRepresentationActive(bool active)
 
             minimapRepresentation.SetActive(active);
         }
+
+        private void OnEntityTransformChanged(DCLTransform.Model updatedModel)
+        {
+            lastAvatarPosition = updatedModel.position;
+
+            UpdateAvatarIconInMinimap(new MinimapMetadata.MinimapUserInfo
+            {
+                userId = model.id,
+                userName = model.name,
+                worldPosition = updatedModel.position
+            });
```

Comment:
Maybe it's a good idea to have only one instance of `MinimapMetadata.MinimapUserInfo` and modify it's properties before calling `UpdateMinimapUserInformation` to avoid that amount of `new MinimapMetadata.MinimapUserInfo`? It's that a possibility?

Reply:
Yes! you're right and that is possible. It's a good idea to have only one instance. I have applied the change. Thanks! :)